.travis.yml

```diff
@@ -54,6 +54,16 @@ matrix:
     env: ZOOKEEPER_VERSION=3.5.4-beta TOX_VENV=py36
   - python: '3.6'
     env: ZOOKEEPER_VERSION=3.5.4-beta TOX_VENV=py36-sasl
+  - python: '3.7'
+    env: ZOOKEEPER_VERSION=3.3.6 TOX_VENV=py37
+  - python: '3.7'
+    env: ZOOKEEPER_VERSION=3.4.13 TOX_VENV=py37 DEPLOY=true
```

Comment:
Sounds good to deploy on pypi for this version, can you please remove the `DEPLOY=true` from L50?

Reply:
Ok